pypeit/images/rawimage.py

```diff
@@ -806,7 +806,9 @@ def flatfield(self, flatimages, slits=None, force=False, debug=False):
         # Generate the illumination flat, as needed
         illum_flat = 1.0
         if self.par['use_illumflat']:
-            illum_flat = flatimages.fit2illumflat(slits, flexure_shift=self.spat_flexure_shift)
+            illum_flat = flatimages.fit2illumflat(slits, spat_flexure=self.spat_flexure_shift)
+            # TODO :: We don't have tilts here yet... might be ever so slightly better, especially on very tilted slits
```

Comment:
Yes I agree with this, we could change the calibration order to have tilts by this point.